src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+    "                           value,\n",
+    "                           units as pyunits)\n",
+    "from pyomo.network import Arc\n",
+    "\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "from idaes.generic_models.properties.core.generic.generic_property import (\n",
+    "        GenericParameterBlock)\n",
+    "from idaes.generic_models.properties.core.generic.generic_reaction import (\n",
+    "        GenericReactionParameterBlock)\n",
+    "from idaes.generic_models.unit_models import (Mixer,\n",
+    "                                              Heater,\n",
+    "                                              PFR)\n",
+    "\n",
+    "from idaes.core.util.constants import Constants\n",
+    "from idaes.core.util import get_solver\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "from idaes.core.util.initialization import propagate_state\n",
+    "\n",
+    "import idaes.logger as idaeslog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing required thermo and reaction package\n",
+    "\n",
+    "The final set of imports are to import the thermo and reaction package. We have created a custom thermo package that support ideal vapor and liquid behavior for this system, and in this case we will restrict it to ideal liquid behavior only.\n",
+    "\n",
+    "The reaction package here assumes Arrhenius kinetic behavior for the PFR, for which $k_0$ and $E_a$ are known *a priori* (if unknown, they may be obtained using one of the parameter estimation tools within IDAES).\n",
+    "\n",
+    "$ r = -kVC_{EO} $, $ k = k_0 e^{(-E_a/RT)}$, with the variables as follows:\n",
+    "\n",
+    "$r$ - reaction rate extent in moles of ethylene oxide consumed per second; note that the traditional reaction rate would be given by $rate = r/V$ in moles per $m^3$ per second  \n",
+    "$k$ - reaction rate constant per second  \n",
+    "$V$ - volume of PFR in $m^3$, note that this is *liquid volume* and not the *total volume* of the reactor itself  \n",
+    "$C_{EO}$ - bulk concentration of ethylene oxide in moles per $m^3$ (the limiting reagent, since we assume excess catalyst and water)  \n",
+    "$k_0$ - pre-exponential Arrhenius factor per second  \n",
+    "$E_a$ - reaction activation energy in kJ per mole of ethylene oxide consumed  \n",
+    "$R$ - gas constant in J/mol-K  \n",
+    "$T$ - reactor temperature in K\n",
+    "\n",
+    "These calculations are contained within the property, reaction and unit model packages, and do not need to be entered into the flowsheet. More information on property estimation may be found below:\n",
+    "\n",
+    "ParamEst parameter estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/workflow/data_rec_parmest.html?highlight=paramest  \n",
+    "HELMET thermodynamic estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/modeling_extensions/surrogate/helmet/index.html  \n",
```

Comment:
I am not sure about referencing HELMET and RIPE right now - the code is an absolute mess, and is not tested so we don't even know if they actually work. Both of those are likely to be rebuilt next year, or pushed of to the side somewhere,

Reply:
Would you recommend that I remove these references from the examples?